config/configtest/configtest.go

```diff
@@ -45,3 +48,15 @@ func NewViperFromYamlFile(t *testing.T, fileName string) *viper.Viper {
 
 	return v
 }
+
+// LoadConfigFile loads a config from file.
+func LoadConfigFile(t *testing.T, fileName string, factories component.Factories) (*configmodels.Config, error) {
```

Comment:
What's the difference in loadConfigFile and LoadConfigFile? Is everything not able to use just LoadConfigFile?

Reply:
circular deps :(, I know is a bit of duplicate code, but cannot do better.